src/wms/src/test/java/org/geoserver/wms/svg/SVGTest.java

```diff
@@ -82,15 +78,26 @@ public void testBatikSvgGenerator() throws Exception {
         assertTrue(doc.getElementsByTagName("g").getLength() > 1);
     }
     
-    @Test
-    public void testBatikMultipleFts() throws Exception {
+    private boolean isw3OrgReachable() {
         //batik includes DTD reference which forces us to be online, skip test
         // in offline case
         try {
-            new URL( "http://www.w3.org").openConnection().connect();
+            HttpURLConnection connection = (HttpURLConnection) new URL("http://www.w3.org")
+                    .openConnection();
+            connection.setConnectTimeout(5000);
+            connection.connect();
+            connection.disconnect();
+            return true;
         } catch (Exception e) {
-            return;
+            e.printStackTrace();
```

Comment:
Given that this feeds into assume, perhaps we could just print the failure on a single line, the stack trace itself if not interesting - only the result that www.w3.org was not available.